lib/rspec/core/did_you_mean.rb

```diff
@@ -0,0 +1,44 @@
+module RSpec
+  module Core
+    # Service object to provide did_you_mean suggestions
+    class DidYouMean
+      attr_reader :relative_file_name
+
+      def initialize(relative_file_name)
+        @relative_file_name = relative_file_name
+      end
+
+      if defined?(::DidYouMean::SpellChecker)
+        # provide probable suggestions if a LoadError
```

Comment:
Nothing checks it's a load error here, so perhaps this comment is unnecessary?

Reply:
done